static/elements/chromedash-new-feature-list.js

```diff
@@ -0,0 +1,86 @@
+import {LitElement, css, html} from 'lit';
+import './chromedash-approvals-dialog';
+import './chromedash-feature-table';
+import SHARED_STYLES from '../css/shared.css';
+
+
+class ChromedashNewFeatureList extends LitElement {
+  static get properties() {
+    return {
+      signedInUser: {type: String},
+      canEdit: {type: Boolean},
+      canApprove: {type: Boolean},
+      loginUrl: {type: String},
+      starredFeatures: {type: Object}, // will contain a set of starred features
+    };
+  }
+
+  constructor() {
+    super();
+    this.signedInUser = ''; // email address
+    this.starredFeatures = new Set();
+    this.canEdit = false;
+    this.canApprove = false;
+  }
+
+  static get styles() {
+    return [
+      SHARED_STYLES,
+      css`
```

Comment:
Is an empty css needed here, or is this just a placeholder?

Reply:
It is just a placeholder.  I suppose that I could delete it, but I kind of like having consistency among all the custom elements.